templates/base.html

```diff
@@ -205,13 +205,16 @@
                         </div>
                     </div>
                     <div class="content">
-                        <!-- ko if: Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.ParameterTable || Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.GraphConfigAttributesTable -->
+                        <!-- ko if: $root.getEagleIsReady() && Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.ParameterTable -->
```

Comment:
Maybe you could remove the four $root.getEagleIsReady() checks, and instead use a single check for a "ko if" block around the whole thing.